tests/compile/piecewise/test_simple.py

```diff
@@ -127,10 +127,13 @@ def test_simple_piecewise_compile(use_inductor):
 
 
 @torch.inference_mode()
-def test_simple_inductor_graph_partition():
+@pytest.mark.parametrize("splitting_ops", [["silly.attention"], []])
+def test_simple_inductor_graph_partition(splitting_ops):
     assert VLLM_USE_V1
     _run_simple_model(
-        splitting_ops=[],
+        # inductor graph partition automatically resets splitting_ops
```

Comment:
This is not true; they get set to empty if they are not set (None by default). I think in this case `silly.attention` will still be a splitting op